olp-cpp-sdk-authentication/include/olp/authentication/AuthenticationCredentials.h

```diff
@@ -59,6 +60,39 @@ class AUTHENTICATION_API AuthenticationCredentials {
    */
   const std::string& GetSecret() const;
 
+  /*
+   * @brief Reads credentials to access OLP from an input stream that is
+   * interpreted as a sequence of characters.
+   *
+   * The stream must contain the following sequences of characters:
+   * here.access.key.id = <user's key id>
+   * here.access.key.secret = <user's key secret>
+   *
+   * @param[in] stream Stream to read the credentials from.
```

Comment:
```suggestion
   * @param[in] stream The stream from which the credentials are read.
```

Reply:
Please check the latest revision (e3a90c4) that was pushed 2 hours ago, instead of penultimate.
You will find that this comment is addressed already.

Reply:
Checked: `   * @param[in] stream Stream to read the credentials from.`
The comment wasn't addressed

Reply:
You're right. Sorry, my bad.
Done.